### DISCUSSION TITLE
Thank you Alex Goodwin!

### DISCUSSION THREAD

**Discussion by shashanknimje** [mcmonkeyprojects/SwarmUI#349] [General]
Hi Alex @mcmonkey4eva ,

I just want to take this moment to thank you very much for the work that you have done with SwarmUI. I am a very new user, and I've found it to be extremely detailed. Still learning how to use it, let alone how complex it must have been coding it. So thank you for all you do Sir! Hope you have a great one!

**Comment by droseraph** [comment]
Thank you too !

**Comment by FurkanGozukara** [comment]
Yes he is a legend

**Comment by dk1953** [comment]
I'll drop my thanks here too: just wanted to say I liked the recent(ish) metadata usability changes allowing for the frontend to read metadata from the "new" json files when placed alongside images without embedded metadata. It was easy enough to just rename my previous library .txt files, so it was a fairly painless transition.

the transition from Nvidia to AMD wasn't so painless, especially with the rocm/whatever stack causing desktop UI , but the SwarmUI setup at least made that part of the setup very easy.

**Comment by CPioGH2002** [comment]
*I was looking for a general appreciation thread and finally found it. :-)* Seems like not that many folks visit the discussion section these days though.

---

It has to be said that this project is a great example for the dev's skill, foresight and overall execution:

- The idea to have ComfyUI in the background (back end) with a simpler interface on top is ingenious and allows for easy and super-nerdy use at the same time. The easy switching back and forth still blows my mind.
- Having continued support for the latest and greatest models with proper tweaks in place and helpful notes from the dev on how to use them is such a treat. **In general, the docs on this project are excellent:** The notes, the tables, the extra links and tips. The devs either love to write them or "just" know how much good documentation is worth. Or both.
- The amount of options and useful(!) tooltips is amazing. The ideas and concepts are well executed and it's a delight to find just that option or setting which I previously thought couldn't already exist.

Since the installation (Linux user here) also worked nicely while the operation is solid and fast, there's much to love and admire here. No negatives so far, using it since months.

PS: I forgot to add that I really appreciate the existence and structure of your UI tooltips. So good!
